locales/en/messages.json

```diff
@@ -5737,7 +5737,7 @@
         "message": "Mass Storage Mode"
     },
     "onboardLoggingMscNote": {
-        "message": "Reboot into <strong>mass storage device (MSC)</strong> mode. Once activated, the onboard flash or SD card on your flight controller will be recognised as a storage device by your computer, and allow you to download your log files. Eject and power cycle your flight controller to leave mass storage device mode."
+        "message": "Reboot into <strong>mass storage device (MSC)</strong> mode. Once activated, the onboard flash or SD card on your flight controller will be recognised as a storage device by your computer, and allow you to download your log files. Eject and power cycle your flight controller to leave mass storage device mode. Return here to erase flash."
```

Comment:
Im not sure about "return here to erase flash" statement, because it also can do using cms/osd menu, or stick combination.

Reply:
Thanks. @ctzsnooze as already approved I'll update anyway because @Asizon found a typo. If you want anything changed please let me now.

Reply:
@Asizon fixed your comments, thank you!

Reply:
Goodpoint @Asizon - maybe we can say, "Return here, or use OSD or stick commands to erase flash".

Reply:
The problem is that the mounted mass storage volume is locked, and files cannot be erased from it using the normal computer methods.  I wastes several hours trying to figure out how to do it in the computer.  This is one reason why I'll be sticking with 'save flash to file' for the foreseeable future.

Reply:
The being shown as locked is just a follow on effect from the fact that in the case of the on-board flash that you are using, we are not storing the logs as actual files, just as a data stream. So the metaphor of 'deleting a file' simply does not apply to this type of storage.